docs/real-user-metrics.md

```diff
@@ -83,16 +83,16 @@ The [icinga alert was originally added in this PR](https://github.com/alphagov/g
 
 You will need to download the new script from [the SpeedCurve github repo](https://github.com/SpeedCurve-Metrics/lux.js). This is a private repo and you will need to request access. Matt Hobbs (head of frontend) is our point of contact for this. Once you have access it's worth turning on notifications for this repo in case the icinga alert fails.
 
-The script is written in TypeScript so you will first need to `npm install` and `npm build` to create a JavaScript version of the file (see repo README for details).
+The script is written in TypeScript so you will first need to `npm install` and `npm run build` to create a JavaScript version of the file (see repo README for details).
 
 ### Updating
 
-Updating the reporter is a case of copying and pasting, reformatting, and updating to include the variables mentioned earlier. Try to format the file so that indenting differences don't make reviewing difficult.
+Update `lux-reporter.js` by copying `dist/lux.js` from the Lux repo. This is a case of copying and pasting, reformatting, and updating to include the variables mentioned earlier. Try to format the file so that indenting differences don't make reviewing difficult.
```

Comment:
Quick question: do you think there would be any mileage in making "Lux repo" a link? Just a thought that occurred to me, happy to leave it as it is if preferable.

Reply:
It's linked a few paragraphs above, but called something else, so I've renamed the text here to be clearer.